tests/pkg_etl/app.config.test

```diff
@@ -0,0 +1,2 @@
+CONFIG_MODULE_CPP=y
```

Comment:
Better use `select MODULE_CPP` under the `config PACKAGE_ETL` definition than here.

```suggestion
```

Reply:
Makes more sense, thanks!